src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -192,33 +201,39 @@ private static javax.net.ssl.HttpsURLConnection writeBody(final javax.net.ssl.Ht
 
     private static javax.net.ssl.HttpsURLConnection createGetConnection(final String url, final String query,
                                                                         final String apiKey) throws IOException {
-        String getURL = String.format("%s?%s", url, query);
-        javax.net.ssl.HttpsURLConnection conn = createEasyPostConnection(getURL, apiKey);
+        String getURL = url;
+        if (query != null) {
+            getURL = String.format("%s?%s", url, query);
+        }
+        javax.net.ssl.HttpsURLConnection conn = createEasyPostConnection(getURL, apiKey, "GET");
         conn.setRequestMethod("GET");
```

Comment:
This line can be removed since we are passing in `method` in the parameter and we are setting the request method in the `createEasyPostConnection` function in line 165